api/views/assets.py

```diff
@@ -149,7 +149,7 @@ def get_queryset(self):
 class AssetLogViewSet(ModelViewSet):
     serializer_class = AssetLogSerializer
     queryset = models.AssetLog.objects.all()
-    permission_classes = [IsSecurityUser]
+    permission_classes = [IsLogUser]
```

Comment:
I think we can do `(IsSecurityUser|IsAdminUser, )` instead of writing a new permission class